### ISSUE TITLE
Accuracy issue, implementation difference compared to paper.

### ISSUE THREAD

**Issue by avinash31d** [clovaai/ext_portrait_segmentation#17]
Hi,

I tried retrain the model on biadu_V2, i am getting training iou as 0.945, but i evaluated on real life images, I am not able to get good results.

Also I can see the loss function you used in here is lovasz whereas the paper [SINET](https://arxiv.org/pdf/1911.09099v4.pdf) has a completely different loss function. Can you provide some information on why the loss function and what could be the problem with these test images?

**Comment by HYOJINPARK** [comment]
@avinash31d 
In here, I used normal CE loss and you can change it by modifying json file

**Comment by avinash31d** [comment]
@HYOJINPARK 

Thanks for taking out time,

I tried CE but it seems lovasz does it better. Also I posted few results just to compare. Can you crop the left side image and run through your model and upload the results here for comparison because I doubt if my model is working.

**Comment by HYOJINPARK** [comment]
@avinash31d  
You can use my model, I uploaded the pth file

**Comment by ZshahRA** [comment]
Hello @avinash31d,

Could you please provide your code for visualize_video.py. I want to see how you test your new images. I have trouble in loading Extremec3Net model in visualize_video.py. Thank you
